package.json

```diff
@@ -27,14 +27,16 @@
     "test:api": "npm run test:api:jest --maxWorkers=50% && concurrently -k -s first \"wait-on http://localhost:3000/explorer/ && npm run test:api:mocha\" \"npm run start\"",
     "test:api:jest": "jest --config ./test/config/jest-e2e.json --maxWorkers=50%",
     "test:api:mocha": "mocha --config ./test/config/.mocharc.json",
-    "prepare:local": "docker-compose -f CI/E2E/docker-compose-local.yaml up -d && cp functionalAccounts.json.example functionalAccounts.json"
+    "prepare:local": "docker-compose -f CI/E2E/docker-compose-local.yaml --env-file CI/E2E/.env.elastic-search up  -d && cp functionalAccounts.json.example functionalAccounts.json"
```

Comment:
I wonder if we didn t want to store the environmental variables in HELM charts, if there is any difference between environments

Reply:
@enthusiastio
We should store the ENV variables in the helm charts for dev environment. This prepare:local script is only for local usage, to spin up containers for local development.